hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.module.scss

```diff
@@ -59,37 +59,39 @@
     }
 }
 
+.noWrap {
+    white-space: nowrap;
+}
 .width {
-    &Fixed {
```

Comment:
Here's the culprit, this is still used in the code a lot

Reply:
oops I didn't mean to delete it

Reply:
it's been added back on line 87